rest/base_client.go

```diff
@@ -64,13 +65,13 @@ type BaseClient struct {
 
 func NewBaseClient(rawHost string, cli akid.ClientID) BaseClient {
 	host := "api." + rawHost
-	// If rawHost is IP, IP:port, localhost, or localhost:port, use that
-	// directly. This is mostly to support tests.
+	// If rawHost is either of these: IP, IP:port, localhost, localhost:port or postman host
+	// use that directly. This is to support postman agent and tests.
 	if h, _, err := net.SplitHostPort(rawHost); err == nil {
 		if h == "localhost" || net.ParseIP(h) != nil {
 			host = rawHost
 		}
-	} else if rawHost == "localhost" || net.ParseIP(rawHost) != nil {
+	} else if rawHost == "localhost" || net.ParseIP(rawHost) != nil || strings.Contains(rawHost, "postman") {
```

Comment:
This is a good idea. I worry that it's a bit too broad.

Maybe what we can do is lift the conversion from 

domain akita.software => host api.akita.software
domain staging.akita.software => host api.staging.akita.software
domain localhost => host localhost

to a higher level where we're simultaneously doing the environment to domain mapping?  We may just be able to scrap the "prefix api" logic here, since it's not really used for anything but akita-staging and akita-prod.  (Originally the thinking was that customers would have their own isolated clusters and so you'd have something like api.customername.akita.software.)

Reply:
That is, the idea is that if the user specifies "staging.akita.software" as the domain then we translate it to the endpoint "api.staging.akita.software" and the same for the special value "akita.software", but have those be the two exceptional cases rather than the current list of exceptions (localhost, IP address, and now "postman" as a substring.)

The logic can treat the two previous cases as the exceptions rather than the ones we want to support going forward.

Reply:
Will pick this up in the next milestone, for now, will only target to make things working for demo.